src/Docker/docker-compose.yml

```diff
@@ -39,6 +39,9 @@ services:
       - AZURE_STORAGE_CONNECTION_STRING=$AZURE_STORAGE_CONNECTION_STRING
       - EMAIL_PASSWORD=$EMAIL_PASSWORD
       - ADMIN_EMAIL=vectoratlas@icipe.org
+      - ANALYTICS_ADMIN_PASSWORD=$VA_ANAL_ADMIN_PASSWORD
+      - ANALYTICS_API_URL=http://localhost:3003/analytics/api
+      - NEXT_PUBLIC_ANALYTICS_ID=fa6b40f4-9ba3-4f80-936c-2696c34f62db
```

Comment:
I think this is probably different for the server

Reply:
> I think this is probably different for the server

This is what I'm wondering. I'm unsure how to setup the env variables so it is compatible with server

Reply:
Is this something we need to keep secret? If so, we'll need to follow the usual secret process. If not we can just stick the actual value in here. You can find the actual value by using DevTools on chrome, so I don't think it's that secret (it's `2b3b6da0-3e75-4d9e-be86-d588496a174e`)

Reply:
The problem is that the ID changes. So the ID I put in will not be the same for the deployed instance. So, in the analytics service I have it as a property that is obtained from an API call - however, I have just noticed that we also use it in _app.tsx

So not sure how want to handle that 

```
function MyApp({ Component, pageProps }: AppProps) {
  useEffect(() => {
    store.dispatch(getFeatureFlags());
    store.dispatch(getMapStyles());
    store.dispatch(getUiVersion());
    store.dispatch(getApiVersion());
    store.dispatch(getTileServerOverlays());
  }, []);
  return (
    <>
      <Script
        async
        defer
        data-website-id={process.env.NEXT_PUBLIC_ANALYTICS_ID}
        src={process.env.NEXT_PUBLIC_ANALYTICS_URL}
      />
      <Provider store={store}>
        <ThemeProvider theme={theme}>
```

Reply:
If you put in the id for the deployed instance it'll be the same - that's the one I copied above. It should persist, as it's in the db. Seeing as we have it in the env variables for the UI, I think we can just stick it in for the API too.